README.md

```diff
@@ -0,0 +1,44 @@
+# nobl9-action
```

Comment:
the name makes me think it will generically support any actions on nobl9, but the description feels a bit more scoped to `sloctl` handling an SLO resource.

What's the long term intent for the action?

Reply:
The intent is to support users of nobl9 in CI/CD, which currently is just creating resources in Nobl9.  The action will take any yaml file for Nobl9 and apply it, not just SLOs.  the `-action` is a convention for repos that hold Github actions.

Reply:
Okie, sounds good to me